src/Channel.cpp

```diff
@@ -86,13 +98,13 @@ JointState Channel::getJointState() const {
         case CONTROL_SPEED:
         case CONTROL_SPEED_POSITION: {
             int32_t feedback = get<Feedback>();
```

Comment:
The point of the source change is that while in AUTO and NONE we keep the current behaviour, in ENCODER we can read it while in OPEN_LOOP mode. Add it to the OPEN_LOOP case.